src/main/java/executor/service/App.java

```diff
@@ -1,24 +1,18 @@
 package executor.service;
 
-import executor.service.config.di.FactoryForDI;
-import executor.service.config.di.ApplicationContext;
-import executor.service.service.ParalleFlowExecutorService;
-
-import java.util.HashMap;
-import java.util.Map;
+import executor.service.DIFactory.ObjectFactoryImpl;
+import executor.service.service.StepExecutionClickCss;
 
 public class App {
 
-    public static void main( String[] args ) {
-        ApplicationContext context = FactoryForDI.run(
-                getPackageName(),
-                new HashMap<>(Map.of(ParalleFlowExecutorService.class, ParalleFlowExecutorService.class)));
-        ParalleFlowExecutorService service = context.getObject(ParalleFlowExecutorService.class);
-        service.execute();
-    }
+  public static void main(String[] args)
+      throws Throwable {
 
-    private static String getPackageName() {
-        Package currentPackage = App.class.getPackage();
-        return currentPackage.getName();
-    }
+    ObjectFactoryImpl factory = ObjectFactoryImpl.getInstance();
```

Comment:
1. Give recommendations on creating classes and contracts.
2. Give recommendations on injecting classes into classes.